Articles/_posts/Medium/Yearn State of the Vaults/The Vaults at Yearn/2021-11-05-The-Vaults-at-Yearn.md

```diff
@@ -4,1082 +4,869 @@ title:  "The Vaults at Yearn"
-This vault accepts deposits of [yDAI+yUSDC+yUSDT+yBUSD](https://etherscan.io/address/0x3B3Ac5386837Dc563660FB6a0937DFAa5924333B) tokens obtained by supplying BUSD, DAI, USDC, USDT, (or their wrapped version yBUSD, yDAI, yUSDC or yUSDT) to the liquidity pool on Curve [here](https://curve.fi/busd/deposit). yDAI+yUSDC+yUSDT+yBUSD tokens are staked in the gauge on Curve to earn CRV rewards. Rewards are swapped for one of the underlying assets and resupplied to the liquidity pool to obtain more yDAI+yUSDC+yUSDT+yBUSD.
+यह तिजोरी [yDAI+yUSDC+yUSDT+yBUSD](https://etherscan.io/address/0x3B3Ac5386837Dc563660FB6a0937DFa5924333B) टोकन की जमाराशियां स्वीकार करती है, जो BUSD, DAI, USDC, USDT, (या उनके लिपटे संस्करण yDAI, USDC, yDAI, USDC  या yUSDT) की आपूर्ति से प्राप्त होती हैं कर्व पर लिक्विडिटी पूल के लिए [यहां](https://curve.fi/busd/deposit)। yDAI+yUSDC+yUSDT+yBUSD टोकन CRV पुरस्कार अर्जित करने के लिए कर्व पर गेज में दांव पर लगे हैं। अंतर्निहित परिसंपत्तियों में से एक के लिए पुरस्कारों की अदला-बदली की जाती है और अधिक yDAI+yUSDC+yUSDT+yBUSD प्राप्त करने के लिए तरलता पूल में पुन: आपूर्ति की जाती है।
 
 [ConvexyDAI+yUSDC+yUSDT+yBUSD](https://etherscan.io/address/0x3cA0B4d7eedE71061B0bAdb4F0E86E99b0FEa613) 🚀  
-Supplies yDAI+yUSDC+yUSDT+yBUSD to Convex Finance to earn CRV and CVX. Rewards are harvested, sold for more yDAI+yUSDC+yUSDT+yBUSD, and deposited back into the vault.
+CRV और CVX अर्जित करने के लिए Convex Finance को yDAI+yUSDC+yUSDT+yBUSD की आपूर्ति करता है। पुरस्कारों को काटा जाता है, अधिक yDAI+yUSDC+yUSDT+yBUSD में बेचा जाता है, और वापस तिजोरी में जमा किया जाता है।
 
-![](70.png)
+![](53.png)
 
-### Curve Compound Pool yVault ([yvCurve-Compound](https://etherscan.io/address/0xD6Ea40597Be05c201845c0bFd2e96A60bACde267))
+### v2 Curve Compound Pool yVault ([yvCurve-Compound](https://etherscan.io/address/0xD6Ea40597Be05c201845c0bFd2e96A60bACde267))
 
 [CurvecDAI+cUSDCVoterProxy](https://etherscan.io/address/0xdDAAc8B5Dd65d079b6572e43890BDD8d95bD5cc3) 🚀  
-This vault accepts deposits of [cDAI+cUSDC](https://etherscan.io/address/0x845838df265dcd2c412a1dc9e959c7d08537f8a2) tokens obtained by supplying DAI or USDC (or their wrapped versions from Compound) to the liquidity pool on Curve [here](https://curve.fi/compound/deposit). cDAI+cUSDC tokens are staked in the gauge on curve.finance to earn CRV rewards. Rewards are swapped for one of the underlying assets and resupplied to the liquidity pool to obtain more cDAI+cUSDC.
+यह तिजोरी [cDAI+cUSDC](https://etherscan.io/address/0x845838df265dcd2c412a1dc9e959c7d08537f8a2) की जमा राशि स्वीकार करती है, जो कर्व पर तरलता पूल में DAI या USDC (या कंपाउंड से उनके लिपटे संस्करण) की आपूर्ति करके प्राप्त की जाती है [यहां](https: //curve.fi/compound/जमा)। cDAI+cUSDC टोकन CRV पुरस्कार अर्जित करने के लिए कर्व.फाइनेंस पर गेज में दांव पर लगे हैं। अंतर्निहित परिसंपत्तियों में से एक के लिए पुरस्कारों की अदला-बदली की जाती है और अधिक cDAI+cUSDC प्राप्त करने के लिए चलनिधि पूल में पुन: आपूर्ति की जाती है।
```

Comment:
fix link dont translate no space in it

Reply:
यह तिजोरी [cDAI+cUSDC](https://etherscan.io/address/0x845838df265dcd2c412a1dc9e959c7d08537f8a2) की जमा राशि स्वीकार करती है, जो कर्व पर तरलता पूल में DAI या USDC (या कंपाउंड से उनके लिपटे संस्करण) की आपूर्ति करके प्राप्त की जाती है [यहां](https: //curve.fi/compound/deposit) ।जमा cDAI+cUSDC टोकन CRV पुरस्कार अर्जित करने के लिए कर्व.फाइनेंस पर गेज में दांव पर लगे हैं। अंतर्निहित परिसंपत्तियों में से एक के लिए पुरस्कारों की अदला-बदली की जाती है और अधिक cDAI+cUSDC प्राप्त करने के लिए चलनिधि पूल में पुन: आपूर्ति की जाती है।